plugins/server_redirect/plugin_info.json

```diff
@@ -0,0 +1,13 @@
+{
+    "id": "server_redircet",
```

Comment:
Plugin id does not match the folder name.

Reply:
fixed